manifests/p/Posit/Quarto/1.3.353/Posit.Quarto.locale.en-US.yaml

```
@@ -0,0 +1,61 @@
+# Created with YamlCreate.ps1 v2.2.5 $debug=QUSU.CRLF.7-3-4.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Posit.Quarto
+PackageVersion: 1.3.353
+PackageLocale: en-US
+Publisher: Posit Software, PBC
```

```suggestion
Publisher: Rstudio, PBC
```
Does this need to be under Rstudio.Quarto? 👀

---

The publisher renamed from RStudio to Posit last year but they forget to modify this entry. I'll make a PR for them later.
https://posit.co/blog/rstudio-is-now-posit/